Automattic/jetpack
# Boost: Update Image Size Analysis to use real data for last updated timestamp

Depends on D115367-code. Merge this PR into trunk after deploying it.

Fixes https://github.com/Automattic/jetpack/issues/31503

## Proposed changes:

* Updates the last_updated property to use data provided by wp.com.

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost with Image Size Analysis;
* Make sure to apply this patch on your sandbox D115367-code;
* Request a report;
* After the report is done, go to the detailed page;
* The `Latest report` should show the correct time now (was previously hard-coded).

<img width="498" alt="CleanShot 2023-07-05 at 21 42 13@2x" src="https://github.com/Automattic/jetpack/assets/11799079/c967732e-2f8b-4ec0-95d5-6cb813da06ed">
